multi-cluster-orchestrator/samples/gemma_vllm/2-workload/charts/gemma-vllm-application/templates/placement.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: orchestra.multicluster.x-k8s.io/v1alpha1
+kind: MultiKubernetesClusterPlacement
+metadata:
+  name: gemma-vllm-placement-autoscale
+spec:
+  rules:
+    - type: "all-clusters"
+    - type: "cluster-name-regex"
+      arguments:
+        regex: "^fleet-cluster-inventory/{{ .Values.clusters_prefix }}-"
+  scaling:
+    autoscaleForCapacity:
+      useDraining: true
```

Comment:
This doesn't need to be here until we're using the new image

Reply:
Yup, good catch, pulled for https://github.com/GoogleCloudPlatform/gke-fleet-management/pull/52#discussion_r2074353222